src/Wallabag/ApiBundle/Controller/SearchRestController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Wallabag\ApiBundle\Controller;
+
+use Hateoas\Configuration\Route;
+use Hateoas\Representation\Factory\PagerfantaFactory;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Pagerfanta\Adapter\DoctrineORMAdapter;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class SearchRestController extends WallabagRestController
+{
+    /**
+     * Search all entries by term.
+     *
+     * @ApiDoc(
+     *       parameters={
+     *          {"name"="term", "dataType"="string", "required"=false, "format"="any", "description"="Any query term"},
+     *          {"name"="page", "dataType"="integer", "required"=false, "format"="default '1'", "description"="what page you want."},
```

Comment:
I would have added a mimimum parameter here but it doesn't seem to be available on NelmioApiDocBundle < 3.x